arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/GeoAggregateRESTService.java

```diff
@@ -775,18 +775,18 @@ public Response geotilegmm(
             // --------------------------------------------------------
             // ---------------------- GMM PARAMETERS ------------------
             // --------------------------------------------------------
-            @ApiParam(name = "abscissaUnit",
+            @ApiParam(name = "abscissaunit",
                     value = Documentation.GMM_ABSCISSA_UNIT)
-            @QueryParam(value = "abscissaUnit") String abscissaUnit,
+            @QueryParam(value = "abscissaunit") String abscissaunit,
```

Comment:
You should keep lowerCamelCase naming for parameters (e.g. @QueryParam(value = "abscissaunit") String abscissaUnit)